docs/cody/usage-and-pricing.mdx

```diff
@@ -47,6 +56,28 @@ To revert back to Cody Free from Pro:
 
 To upgrade from Cody Pro to Cody Enterprise, you should [Contact Sales](https://sourcegraph.com/contact/request-info) and connect with one of our account teams. They will help you set up your account and start with Cody Enterprise.
 
+## Billing FAQs for Cody Pro
+
+### What is Cody's pricing plan?
+
+You can get this info from our [pricing page](https://sourcegraph.com/pricing?product=cody).
+
+### When are payments taken?
+
+We charge payments at the beginning of each billing cycle, and they get automatically renewed once you've added your credit card details.
+
+### What is a billing cycle?
+
+The billing cycle refers to the start date and depends on your account creation date. You can view your current and previous billing cycles from the **My subscription** tab.
+
+### What payment methods are available?
+
+We only support payments through a credit card.
+
+### What happens when I cannot pay or don't want to renew the Pro plan?
+
+If you are on a Pro plan trial, you must add a credit card to your account to continue on a Pro plan. If you do not want to renew Cody Pro, you will automatically downgrade to Cody Free.
```

Comment:
We should remove "If you are on a Pro plan trial, you must add a credit card to your account to continue on a Pro plan" on 2/21 because trials will no longer a thing anymore. 

I think we should have some information about cancelling their plan. So something like:

- If you do not want to renew Cody Pro, you can cancel your Cody Pro subscription at any time by going to accounts.sourcegraph.com/cody/subscriptions. If you cancel your Cody Pro plan, your Cody Pro plan will continue until the end of the billing cycle, at which point your plan will not renew, your card will not be charged, and your plan will change from Cody Pro to Cody Free. 
- If you change your mind after cancelling your plan and you have a credit card on file, you can simply click "Resume subscription" on accounts.sourcegraph.com/cody/subscriptions to re-upgrade to Cody Pro.